test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
How about `docker-library-test/*`? Then again, using un-namespaced names has the benefit that it can't conflict since it could only conflict with another official image :+1: